content/_data/changelogs/lts.yml

```diff
@@ -2808,6 +2808,35 @@
       message: >
         Internal: <code>hudson.util.ProcessTree.OSProcess#getEnvironmentVariables</code> returned <code>null</code> when an error occurred even though it shouldn't.
 
+- version: 2.190.3
+  date: 2019-11-20
+  changes:
+    - type: bug
+      pull: 4223
+      issue: 59514
+      message: >
+        Robustness: Do not allow users to resubmit requests using POST on URLs requiring a form submission, as that will fail anyway.
+    - type: bug
+      pull: 4228
+      issue: 56809
+      message: The <code>lastCompletedBuild</code> permalink was not being cached in the <code>…/builds/permalinks</code> file.
+    - type: bug
+      pull: 4269
+      issue: 48375
+      message: Fix labels to Atom feed links.
+    - type: bug
+      pull: 4246
+      references:
+      - issue: 58296
+      - issue: 53462
+      - url: https://bugzilla.mozilla.org/show_bug.cgi?id=1370630
+        title: Firefox issue 1370630
+      message: |-
+        Revert changes in forms submission in Jenkins classic UI with Firefox.
+        Changes caused a regression on forms with "file" inputs.
+        These were made to anticipate a bugfix in Firefox which has been backed out since.
+        (regression in 2.173)
```

Comment:
FWIW with version references like this I try to figure out the equivalent LTS release, and specify that here instead, based on the assumption users are either on the weekly, or on LTS, and so a weekly version reference in the LTS changelog is not helpful.

Reply:
Thanks.  To be sure that I understand, it should have referred to "regression in 2.176.1" as the first LTS version which included the regression instead of referring to "regression in 2.173"?

Reply:
Neither. We backported it to 2.164.3, so that's the version I would mention here.

Reply:
Thanks.  Attempt to fix the mistake in https://github.com/jenkins-infra/jenkins.io/pull/2672